config/crd/bases/cluster.kubernetes-fleet.io_memberclusters.yaml

```diff
@@ -100,6 +100,9 @@ spec:
                   name:
                     description: Name of the object being referenced.
                     type: string
+                    x-kubernetes-validations:
```

Comment:
Why use this approach and not this?
OpenAPIv3 value validations (maxLength, maxItems, maxProperties, required, enum, minimum, maximum, ..) and [string formats](https://swagger.io/docs/specification/data-models/data-types/#format) where available.

also shown in [k8s docs](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)